cumulus/variables.tf

```diff
@@ -270,6 +270,12 @@ variable "urs_url" {
   default     = "https://uat.urs.earthdata.nasa.gov"
 }
 
+variable "deploy_cumulus_distribution" {
+  description = "If true, does not deploy the TEA distribution API"
+  type        = bool
+  default     = true
```

Comment:
I'll approve this PR as soon as the default value is false as it is in Cumulus

Reply:
Sounds good. I'll have some time today to catch up on all these outstanding PR's

Reply:
@reweeden You'll see that I merged the CHANGELOG entry from prod into this branch to resolve a merge conflict due to several unreleased updates.  Once we get them all in place, I'll create a point release of CIRRUS-Core.  I'd like to do that before starting on Cumulus v18.